### PR TITLE
Just fixed some typos in the readme.

### PR DESCRIPTION
**Pull request by justgerd**

### CODE REVIEW

**Comment by justgerd** [comment]
And I added a section on how to run the bot.

File: README.md [hunk 1]
```diff
@@ -6,10 +6,14 @@ i currently suck at python, and i'm not really thinking too much, so this code i
 
 ## Installing 
 
-`pip install -r requirments.txt`
+`pip install -r requirements.txt`
```
Comment (nullpixel) [inline README.md:9]: you need pip3.

File: README.md [hunk 2]
```diff
@@ -6,10 +6,14 @@ i currently suck at python, and i'm not really thinking too much, so this code i
...
 
 add credentials to config.py (rename or copy config.example.py to config.py ).
 
+## Running
+
+`python bot.py`
```
Comment (nullpixel) [inline README.md:15]: `python3 bot.py`

File: README.md [hunk 3]
```diff
@@ -18,4 +22,4 @@ it'll include an optional mysql logging cog too, yay for nsa.
 
 ## config
 
-in each guild, the only requiered log channel is `server-log`. if, for example, `message-log` is missing it will default to `server-log`.
\ No newline at end of file
+in each guild, the only requiered log channel is `server-log`. if, for example, `message-log` is missing it will default to `server-log`.
```
Comment (nullpixel) [inline README.md:25]: it now only logs to one log channel, set by `!setlog`

**Review by nullpixel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xela-zone) [reply README.md:9]: not if all you have is python3 :mmlol:

File: README.md (hunk 2 above)
Comment (xela-zone) [reply README.md:15]: or ./bot.py on linux

Comment (nullpixel) [reply README.md:15]: and macOS

File: README.md (hunk 3 above)
Comment (xela-zone) [reply README.md:25]: i should have changed this before we merged develop -> master

File: README.md (hunk 1 above)
Comment (nullpixel) [reply README.md:9]: mhm, too many goddamn things

guess i can close this

**Comment by nullpixel** [comment]
nice pr, but a lot's wrong with it. lmk if you fix

File: README.md (hunk 1 above)
Comment (justgerd) [reply README.md:9]: still. You can't just copy-pasta this. Also instructions for the db would be nice...

**Pull request closed without merging**